util/vendor.py

```diff
@@ -340,7 +350,8 @@ def __init__(self, handle):
 
 class Desc:
     '''A class representing the configuration file'''
-    def __init__(self, handle):
+
+    def __init__(self, handle, desc_overrides=[]):
```

Comment:
I don't think `desc_overrides` should have a default: we only call the `Desc` constructor in one place, and it's got both arguments.